cmd/olm/manager.go

```diff
@@ -53,6 +57,36 @@ func Manager(ctx context.Context, debug bool) (ctrl.Manager, error) {
 		MetricsBindAddress: "0", // TODO(njhale): Enable metrics on non-conflicting port (not 8080)
 		Cache: cache.Options{
 			ByObject: map[client.Object]cache.ByObject{
+				&appsv1.Deployment{}: {
+					Label: labels.SelectorFromValidatedSet(map[string]string{install.OLMManagedLabelKey: install.OLMManagedLabelValue}),
+				},
```

Comment:
Nit/question: at this point, is it possible/does it make sense to make this the default selector for everything we cache? And then one-off selectors for objects that don't have this label?

Reply:
Is that a suggestion for a change? As of now at least to me it seems like the code in the PR does exactly that.

Reply:
In controller-runtime, you can say "use this selector for ALL objects if there is not an object-specific selector". So I'm wondering if we can do that instead of explicitly listing the same selector for each kind.

Reply:
Ah - sorry. I think they may have gotten rid of that? I can't remember. Let me look.